mako_rollup/src/main.rs

```diff
@@ -18,7 +29,24 @@ fn main() {
 
         // We only care about files, so we intentionally do nothing with directories
         if metadata.file_type().is_file() {
-            let account_uuid = &entry.path().parent().unwrap().to_str().unwrap()[7..];
+            let account_uuid;
```

Comment:
I think it's unusual Rust style to declare ahead of the point of use like this

Reply:
Yep. I like what I've got now better thanks to your input.